src/commands/locks/list.ts

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2022, Nuance, Inc. and its contributors.
+ * All rights reserved.
+ *
+ * This source code is licensed under the Apache-2.0 license found in
+ * the LICENSE file in the root directory of this source tree.
+ */
+
+import chalk from 'chalk'
+import {flags} from '@oclif/command'
+import makeDebug from 'debug'
+
+import {defaultLimit} from '../../utils/constants'
+import * as MixFlags from '../../utils/flags'
+import * as ProjectsAPI from '../../mix/api/projects'
+import {DomainOption} from '../../utils/validations'
+import {MixClient, MixResponse, MixResult,  ProjectsLockListParams} from '../../mix/types'
+import MixCommand from '../../utils/base/mix-command'
+
+const debug = makeDebug('mix:commands:locks:list')
+
+export default class LocksList extends MixCommand {
+  static description = `list project locks
+
+Use this command to get a list of all project locks.
+The list can be constrained using the project, organization
+and/or user flags.
+A project cannot be edited while it is locked.`
+
+  static examples = [
+    'list project locks',
+    '$ mix locks:list -P 249 -O 33 -U 32',
```

Comment:
Let's remove the -O from the example as having the projectID supersedes the organizationID.